tests/HTTP/ClientTest.php

```diff
@@ -221,7 +222,11 @@ public function testSendToGetLargeContent()
         $response = $client->send($request);
 
         $this->assertEquals(200, $response->getStatus());
-        $this->assertLessThan((int) $maxPeakMemoryUsage, memory_get_peak_usage());
+        $this->assertLessThan(
+            (int) $maxPeakMemoryUsage,
+            memory_get_peak_usage(),
+            "Hint: you can adjust the max peak memory usage allowed for this test by defining env variable $maxPeakMemoryUsageEnvVariable to be the desired max bytes"
```

Comment:
```suggestion
            "Hint: you can adjust the max peak memory usage allowed for this test by defining env variable SABRE_HTTP_TEST_GET_LARGE_CONTENT_MAX_PEAK_MEMORY_USAGE to be the desired max bytes"
```

Reply:
The value of `maxPeakMemoryUsageEnvVariable` appears in the string at run-time:
```
phil@phil-Inspiron-5468:~/git/sabre-io/http$ export SABRE_HTTP_TEST_GET_LARGE_CONTENT_MAX_PEAK_MEMORY_USAGE=1000000
phil@phil-Inspiron-5468:~/git/sabre-io/http$ composer phpunit
> phpunit --configuration tests/phpunit.xml
PHPUnit 9.5.21 #StandWithUkraine

Warning:       Your XML configuration validates against a deprecated schema.
Suggestion:    Migrate your XML configuration using "--migrate-configuration"!

.............................F.................................  63 / 159 ( 39%)
............................................................... 126 / 159 ( 79%)
.................................                               159 / 159 (100%)

Time: 00:01.496, Memory: 67.99 MB

There was 1 failure:

1) Sabre\HTTP\ClientTest::testSendToGetLargeContent
Hint: you can adjust the max peak memory usage allowed for this test by defining env variable SABRE_HTTP_TEST_GET_LARGE_CONTENT_MAX_PEAK_MEMORY_USAGE to be the desired max bytes
Failed asserting that 39819328 is less than 1000000.

/home/phil/git/sabre-io/http/tests/HTTP/ClientTest.php:228
phpvfscomposer:///home/phil/git/sabre-io/http/vendor/phpunit/phpunit/phpunit:97

FAILURES!
Tests: 159, Assertions: 293, Failures: 1.
Script phpunit --configuration tests/phpunit.xml handling the phpunit event returned with error code 1
```